package.json

```diff
@@ -53,6 +53,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
+    "test:scss": "sass-lint -v -q",
     "test:sass": "sass-lint -v -q",
```

Comment:
can you remove the old script `"test:sass": "sass-lint -v -q",` ?

Reply:
yeah but without that it was giving error in the checks online tests and also in the local when one enters npm run test:sass

Reply:
got it.. we need to update the tests too. Let's merge this and we'll update the rest in another PR

Reply:
Ok, looking forward to it.